### PR TITLE
AB#39 adding logs to the UserCommandService framework layer

### PR DESCRIPTION
**Pull request by huascraft**
I added the four levels of logs that we are using to the framework layer in usercomanservice

### CODE REVIEW

File: cqrs-user-command-service/src/main/java/dev/amir/usercommand/framework/output/sql/adapter/UserOutputAdapter.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public boolean delete(String userId) {
 
         if (userToDelete.isPresent()) {
             jpaRepository.deleteById(userId);
-            log.info("User deleted with ID: {}", userId);
+            log.info("Eliminating user");
```
Comment (Ramir0) [inline cqrs-user-command-service/src/main/java/dev/amir/usercommand/framework/output/sql/adapter/UserOutputAdapter.java:37]: Replace `Eliminating` with `Deleting`

**Review by Ramir0 — CHANGES_REQUESTED** [review]

File: cqrs-user-command-service/src/main/java/dev/amir/usercommand/framework/output/sql/adapter/UserOutputAdapter.java (hunk 1 above)
Comment (Ramir0) [reply cqrs-user-command-service/src/main/java/dev/amir/usercommand/framework/output/sql/adapter/UserOutputAdapter.java:37]: @huascraft could you fix this?

Comment (huascraft) [reply cqrs-user-command-service/src/main/java/dev/amir/usercommand/framework/output/sql/adapter/UserOutputAdapter.java]: done

File: cqrs-user-command-service/src/main/java/dev/amir/usercommand/framework/input/rest/controller/UserController.java [hunk 2]
```diff
@@ -22,22 +25,38 @@ public class UserController {
...
-
-        return ResponseEntity.ok().build();
+        log.info("Received request to update");
+
+        try {
+            commandHandler.handle(command, id);
+            log.info("User with ID {} has been successfully update", id);
+            return ResponseEntity.ok().build();
+        } catch (Exception e) {
+            log.error("Error updating user with ID {}: {}", id, e.getMessage(), e);
```
Comment (Ramir0) [inline cqrs-user-command-service/src/main/java/dev/amir/usercommand/framework/input/rest/controller/UserController.java:44]: .getMessage() is not needed

**Review by Ramir0 — CHANGES_REQUESTED** [review]

**Review by Ramir0 — APPROVED** [review]

**Pull request merged**